x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -400,6 +408,43 @@ export const ModelsList: FC<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  /**
+   * Updates model list with download status
+   */
+  const fetchDownloadStatus = useCallback(
+    async (fetchInProgress = new Set<string>()) => {
+      try {
+        const downloadStatus = await trainedModelsApiService.getModelsDownloadStatus();
+
+        setItems((prevItems) => {
+          return prevItems.map((item) => {
+            const newItem = { ...item };
+            if (downloadStatus?.[item.model_id]) {
```

Comment:
Nit, after recent change.
```suggestion
            if (downloadStatus.[item.model_id]) {
```

Reply:
Updated in https://github.com/elastic/kibana/pull/184906/commits/d0db2fb9c8d78514a2f7cf02a8487c07150b2568